services/121-service/test/helpers/registration.helper.ts

```diff
@@ -150,12 +150,13 @@ export async function waitForDeleteRegistrations({
     // Get payment transactions
     let totalRegistrationSuccesfullyDeleted = 0;
     for (const referenceId of referenceIds) {
-      const getEventResponse = await getEvents(
+      const getEventResponse = await getEvents({
         programId,
-        undefined,
-        undefined,
+        fromDate: undefined,
+        toDate: undefined,
         referenceId,
-      );
+        accessToken: await getAccessToken(),
```

Comment:
This will (still) do a login-request for EVERY reference-id in the `referenceIds`-array! 😮

We could do this once (at line 148, for example) only...